docs/2-Deploying/2-Configuration.md

```diff
@@ -1240,6 +1240,73 @@ AWS STS role ARN to use for S3 authentication.
   	sts_assume_role_arn: arn:aws:iam::1234567890:role/some-s3-role
   ```
 
+##### Vulnerabilities
+
+###### `databases_path`
+
+The path specified needs to exist and fleet needs to be able to read and write to and from it. This is the only mandatory configuration needed for vulnerability processing to work.
+
+- Default value: none
+- Environment variable: `FLEET_VULNERABILITIES_DATABASES_PATH`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	databases_path: /some/path
+  ```
+
+###### `periodicity`
+
+How often vulnerabilities are checked.
+
+- Default value: `1hr`
+- Environment variable: `FLEET_VULNERABILITIES_PERIODICITY`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	vulnerabilities_periodicity: 1hr
```

Comment:
This shouldn't have the `vulnerabilities_` and the same is true for the ones below.

Reply:
Ah thank you, I did not update the examples.